### PR TITLE
update swap copy

### PR DESCRIPTION
**Pull request by dprats**
update copy for SNS in the dev docs

### CODE REVIEW

**Review by harpa12 — APPROVED** [review]

File: how-it-works/3_governance/00_tokenomics.card.md [hunk 1]
```diff
@@ -6,6 +6,6 @@ title: Tokenomics
 
 # Tokenomics
 
-The Internet Computer makes use of an utility token called ICP. The ICP token is used for a variety of functions on the platform. Users can stake ICP giving them the right to vote and earn voting rewards. Investors can use ICP to invest in SNS sales launched on the Internet Computer, i.e., the initial offerings of the native DAOs. Developers use ICP token to purchase cycles to power their dapps on the Internet Computer. And node providers are remunerated in ICP for the compute power they provide to the Internet Computer platform. 
+The Internet Computer makes use of an utility token called ICP. The ICP token is used for a variety of functions on the platform. Users can stake ICP giving them the right to vote and earn voting rewards. Investors can use ICP to invest in SNS swaps launched on the Internet Computer, i.e., the initial offerings of the native DAOs. Developers use ICP token to purchase cycles to power their dapps on the Internet Computer. And node providers are remunerated in ICP for the compute power they provide to the Internet Computer platform.
```
Comment (georgi-dfinity) [inline how-it-works/3_governance/00_tokenomics.card.md:9]:
```suggestion
The Internet Computer makes use of an utility token called ICP. The ICP token is used for a variety of functions on the platform. Users can stake ICP giving them the right to vote and earn voting rewards. Investors can use ICP to participate in SNS swaps launched on the Internet Computer, i.e., the initial offerings of the native DAOs. Developers use ICP token to purchase cycles to power their dapps on the Internet Computer. And node providers are remunerated in ICP for the compute power they provide to the Internet Computer platform. 
```

File: how-it-works/3_governance/00_tokenomics.subpage.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ slug: tokenomics
...
+The Internet Computer (IC) makes use of a utility token called ICP. This token is used for the following functions on the platform: First, any ICP holder can participate in the governance of the IC by staking ICP in order to vote on proposals and earn voting rewards. Second, you can transform ICP into cycles, which are used as fuel for computation. Third, ICP tokens are used to pay the entities who provide compute capacity by operating node machines. Last but not least, ICP can be used in order to invest in token swaps of decentralized autonomous organizations (DAOs) on the IC. We will elaborate on these four use cases in the following. In addition to the aforementioned platform use cases, ICP can also be used as a medium of exchange, i.e., to pay for goods and services like NFTs, subscriptions, etc.
```
Comment (georgi-dfinity) [inline how-it-works/3_governance/00_tokenomics.subpage.md:13]:
```suggestion
The Internet Computer (IC) makes use of a utility token called ICP. This token is used for the following functions on the platform: First, any ICP holder can participate in the governance of the IC by staking ICP in order to vote on proposals and earn voting rewards. Second, you can transform ICP into cycles, which are used as fuel for computation. Third, ICP tokens are used to pay the entities who provide compute capacity by operating node machines. Last but not least, ICP can be used in order to participate in token swaps of decentralized autonomous organizations (DAOs) on the IC. We will elaborate on these four use cases in the following. In addition to the aforementioned platform use cases, ICP can also be used as a medium of exchange, i.e., to pay for goods and services like NFTs, subscriptions, etc.
```

File: roadmap/4_governance_nns/deployed/1_sns.md [hunk 3]
```diff
@@ -4,10 +4,10 @@ links:
   Proposal: https://dashboard.internetcomputer.org/proposal/65132
   Deployment & Upgrades: https://forum.dfinity.org/t/sns-deployment-and-upgrades-design-proposal/10816
   Rewards: https://forum.dfinity.org/t/open-governance-canister-for-sns-design-proposal/10224/36
-  Decentralization sale: https://forum.dfinity.org/t/sns-initial-token-swap/13591
+  decentralization swap: https://forum.dfinity.org/t/sns-initial-token-swap/13591
```
Comment (georgi-dfinity) [inline roadmap/4_governance_nns/deployed/1_sns.md:7]:
```suggestion
  Decentralization Swap: https://forum.dfinity.org/t/sns-initial-token-swap/13591
```

File: roadmap/4_governance_nns/deployed/1_sns.md [hunk 4]
```diff
@@ -4,10 +4,10 @@ links:
...
 eta: February 2023
 is_community: false
 ---
 
-This SNS rollout will include several features: **1.** SNSs that are provided as a system function (deployed on an SNS subnet and facilitating maintainable upgrades), **2.** A first version of voting rewards for SNSs that can be further customised in the future, **3.** Decentralization sales that decentralize a dapp, where participants provide ICP tokens in exchange for SNS tokens. **4.** A NNS frontend dapp extension that allows end users to participate in the decentralization sale. **5.** Tooling to help users initialize an SNS.
+This SNS rollout will include several features: **1.** SNSs that are provided as a system function (deployed on an SNS subnet and facilitating maintainable upgrades), **2.** A first version of voting rewards for SNSs that can be further customised in the future, **3.** decentralization swaps that decentralize a dapp, where participants provide ICP tokens in exchange for SNS tokens. **4.** A NNS frontend dapp extension that allows end users to participate in the decentralization swap. **5.** Tooling to help users initialize an SNS.
```
Comment (georgi-dfinity) [inline roadmap/4_governance_nns/deployed/1_sns.md:12]:
```suggestion
This SNS rollout will include several features: **1.** SNSs that are provided as a system function (deployed on an SNS subnet and facilitating maintainable upgrades), **2.** A first version of voting rewards for SNSs that can be further customised in the future, **3.** Decentralization swaps that decentralize a dapp, where participants provide ICP tokens in exchange for SNS tokens. **4.** A NNS frontend dapp extension that allows end users to participate in the decentralization swap. **5.** Tooling to help users initialize an SNS.
```

File: src/pages/sns/faq.tsx [hunk 5]
```diff
@@ -675,7 +675,7 @@ function SnsFaqPage() {
                 decentralizing the dapp. In any case, it is crucial that you
                 trust the developers will not change the dapp during the SNS
                 launch. Learn what you could verify and what to look out for{" "}
-                <Link href="https://wiki.internetcomputer.org/wiki/SNS_decentralization_sale_trust">
+                <Link href="https://wiki.internetcomputer.org/wiki/SNS_decentralization_swap_trust">
```
Comment (georgi-dfinity) [inline src/pages/sns/faq.tsx:678]: Please confirm.

File: src/pages/sns/faq.tsx [hunk 6]
```diff
@@ -711,7 +711,7 @@ function SnsFaqPage() {
                 </li>
                 Learn more about verification here: {" "}
                 <Link
-                  href="https://wiki.internetcomputer.org/wiki/How-to:_Verify_SNS_decentralization_sale_proposal"
+                  href="https://wiki.internetcomputer.org/wiki/How-to:_Verify_SNS_decentralization_swap_proposal"
```
Comment (georgi-dfinity) [inline src/pages/sns/faq.tsx:714]: Please confirm.

Comment (dprats) [reply src/pages/sns/faq.tsx:714]: confirmed

File: src/pages/sns/faq.tsx (hunk 5 above)
Comment (dprats) [reply src/pages/sns/faq.tsx:678]: confirmed

**Pull request merged**